MonoGame.Framework.Content.Pipeline/Graphics/Font/GlyphPacker.cs

```diff
@@ -35,10 +37,18 @@ public static BitmapContent ArrangeGlyphs(Glyph[] sourceGlyphs, bool requirePOT,
 			int outputWidth = GuessOutputWidth(sourceGlyphs);
 			int outputHeight = 0;
 
-			// Choose positions for each glyph, one at a time.
+            // Choose positions for each glyph, one at a time.
+            // Keep a record of glyphs in a dictionary so we can look them up based on
+            // their index later for some optimisations
```

Comment:
Hi @PhantomBadger 

I don't follow how looking it up by index in `Dictionary<int, ArrangedGlyph>` vs by index in `List<ArrangedGlyph>` changes things.

Reply:
You're completely right, I think I had some other functionality around the hash-based lookup in a previous implementation that has since been lost. 

Sorry for the confusion there, this was originally intended as a local solution to a problem but after seeing the perf increase I wanted to share it :) I'll update it appropriately in a follow-up commit.